pkg/ccl/changefeedccl/metrics.go

```diff
@@ -1059,14 +1059,14 @@ func newAggregateMetrics(histogramWindow time.Duration, lookup *cidr.Lookup) *Ag
 			Duration:     histogramWindow,
 			MaxVal:       commitLatencyMaxValue.Nanoseconds(),
 			SigFigs:      1,
-			BucketConfig: metric.BatchProcessLatencyBuckets,
+			BucketConfig: metric.LongRunning60mLatencyBuckets,
```

Comment:
since we're making changes to the histogram buckets, can we change them to ones with a min < 500ms? 

also may want to call out which metrics are changing by name in the release notes, in case this affects metric users.